src/components/graph/MMEGraph.js

```diff
@@ -56,19 +64,80 @@ export default class MMEGraph extends Component {
     return maxValue;
   }
 
+  getStats(data) {
+    if (!data || !data.length) return [];
+    //look in data points up to today
+    const copyData = (data)
+    .map(item => { return {...item}})
+    .filter(item=>!((daysFromToday(item[xFieldName]) < 0))) //not future dates
+    if (!copyData.length) {
+        return [];
+    }
+    //get all available dates
+    let arrDates = (copyData.map(item => item[xFieldName]));
+    arrDates = arrDates.filter((d, index) => {
+      return arrDates.indexOf(d) === index;
+    });
+    arrDates = arrDates.map(item => {
+      let returnObject = {};
+      let pointDate = item;
+      returnObject[xFieldName] = pointDate;
+      returnObject[yFieldName] = Math.max(...copyData.filter(o=>o[xFieldName] === pointDate).map(o=>o[yFieldName]));
+      return returnObject;
+    }).sort((a,b) => dateCompare(a[xFieldName], b[xFieldName])); //sort in ascending order
+    //data points for the last 60 days
+    const arrSixtyDays = arrDates.filter(item => {
```

Comment:
Might want to make the length of these intervals (1/60/90 days) function arguments, then they're only hardcoded in one place